lucet-module-data/src/functions.rs

```diff
@@ -21,12 +21,13 @@ impl FunctionIndex {
 }
 
 /// ImportFunction describes an internal function - its internal function index and the name/module
-/// pair that function should be found in.
+/// pair that function should be found in, as well as the name of a hostcall it maps to.
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, Serialize, Deserialize)]
 pub struct ImportFunction<'a> {
     pub fn_idx: FunctionIndex,
     pub module: &'a str,
-    pub name: &'a str
+    pub name: &'a str,
+    pub mapped_to: Option<&'a str>,
```

Comment:
I think `symbol` would be a better name than `mapped_to`. Also, we only ever construct the `Some` case in lucetc/src/decls.rs, and if we encounter a `None` case in DlModule we throw an error. Can we just get rid of the Option for now?

Reply:
`symbol` is good. After our conversation today, I'll make this just `&'a str` and we'll see out that pans out later. Got ahead of myself thinking about how module imports would look.

Reply:
Since this was only present to support the load-time import fixing, and that works by relocation now, this field's just gone